spec/requests/providers/applicants_controller_spec.rb

```diff
@@ -59,11 +59,6 @@
       )
     end
 
-    it "redirects to next page" do
```

Comment:
Should we still test that it redirects, rather than re-render?

Does it add any value?